config/configUtil.js

```diff
@@ -73,17 +73,20 @@ function csvToArray(str) {
  *  replicas.
  */
 function getReadReplicas(pe, replicaLabel) {
-  let replicas;
+  let replicas = [];
   if (pe[replicaLabel]) {
     const replicaList = csvToArray(pe[replicaLabel]);
-    replicas = [];
     replicaList.forEach((replica) => {
       if (pe[replica]) {
         replicas.push(pe[replica]);
       }
     });
   }
 
+  if (!replicas.length) {
+    replicas = undefined;
+  }
+
   return replicas;
```

Comment:
If you just say replicas = undefined at line 76 and do not make anything other changes, it should take care i think.